tasks/ast_tools/src/markers.rs

```diff
@@ -64,7 +64,10 @@ pub struct VisitMarkers {
 /// A struct representing `#[scope(...)]` markers
 #[derive(Default, Debug)]
 pub struct ScopeMarkers {
+    /// `#[scope(enter_before)]`
```

Comment:
Unintentional commented out code?

Reply:
I believe this is intentional. It's a doc comment explaining what attribute `ScopeMarkers::enter_before` represents.